indexer/services/vulcan/src/handlers/order-replace-handler.ts

```diff
@@ -69,7 +85,25 @@ export class OrderReplaceHandler extends Handler {
     const oldOrderId: IndexerOrderId = orderReplace.oldOrderId!;
 
     /* Remove old order */
-    const removeOrderResult: RemoveOrderResult = await this.removeOldOrder(oldOrderId);
+    let removeOrderResult: RemoveOrderResult = { removed: false };
+    try {
+      removeOrderResult = await this.removeOldOrder(oldOrderId, headers);
+    } catch (error) {
+      if (error instanceof ParseMessageError) {
+        return;
+      }
+    }
+
+    /* We don't want to fail if old order is not found (new order should still be placed),
+    so log and track metric */
+    if (!removeOrderResult.removed) {
+      logger.info({
+        at: 'OrderReplaceHandler#handle',
+        message: 'Old order not found in cache',
+        oldOrderId,
+      });
+      stats.increment(`${config.SERVICE_NAME}.replace_order_handler.old_order_not_found_in_cache`, 1);
```

Comment:
should we return here?

Reply:
No, i still want to place the new order if the old order wasn't found